### PR TITLE
Feature: Greek Translation

### PR DESCRIPTION
**Pull request by vmavromatis**
Soundtrack of this PR: [Link](https://www.youtube.com/watch?v=kG12C1oX5Eo)

### Motivation

To provide a full Greek translation.

### In this PR

Translated the text from English to Greek in `/locales/elGR/translation.json`, as well as provided all required changes similar to the zhCN translation so as to show in menu etc 

### Caveats

app.html NEEDS to carry the `lang="el"` tag else the accents look bad in all uppercasing scenarios (see i.e. https://stackoverflow.com/questions/28783259/how-do-i-make-text-transformuppercase-work-properly-with-greek). I did this manually because I was not able to find a way to inject this inside html but I think the proper way to do it is to make this dynamic. (interestingly keeping it as `lang="el"` does NOT break the enUS wallet)

### Future Work

Once the English text is updated, this will need some redo

### Screen Shots
image
image
image

### CODE REVIEW

File: app/app.html [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <!DOCTYPE html>
-<html>
+<!-- HACK: Needs lang tag to ensure accents show properly for capital letters. -->
+<html lang="el">
```
[the remark this answers is not in the document]
Comment (davidyhall) [reply app/app.html:3]: Understood!

We've finalizing the latest feature branch soon enough. It'll have plenty of new copy. Let's keep in touch!

**Comment by vmavromatis** [comment]
Rebased successfully and update translation json to reflect latest changes @davidyhall  
Once the lang tag is implemented (which should be trivial) this can be merged

**Comment by davidyhall** [comment]
Sounds great! I think the team may be moving to a more formal Git Flow. They should merge this into the development branch once it's up and running. Thank you for contributing! If you haven't yet, please make sure to fill out our CLA. https://github.com/mobilecoinofficial/desktop-wallet/blob/main/CLA.md

**Comment by vmavromatis** [comment]
Dear team @davidyhall @voloshyn is this not getting merged? I see that it's closed?

**Pull request closed without merging**